tests/test_logic.py

```diff
@@ -69,7 +69,7 @@ def process(self, instance):
 
 def test_register_gui():
     """Registering at run-time takes precedence over those from environment"""
-    
+
     with no_guis():
         os.environ["PYBLISHGUI"] = "second,third"
```

Comment:
I have never actually used ```PYBLISHGUI```. When was this introduced?
Should this be ```PYBLISH_GUI```instead?
Also finding is odd to use comma instead of os path separator, when working with environment variables.